src/templates/Vue InstantSearch/package.json

```
@@ -3,36 +3,34 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "start": "cross-env NODE_ENV=development webpack-dev-server --port 3000 --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
-    "lint": "eslint --ext .js,.vue .",
+    "start": "vue-cli-service serve",
```

Let's change this for consistency with other templates and to align with the README instructions.

```suggestion
    "start": "vue-cli-service serve --port 3000",
```

---

I've changed it locally, but I discover a conflict with Prettier on my local setup. The service lint correctly but not the editor. Do you any issue?

---

I'm not sure I understand. What are the steps to reproduce?

---

Whenever I save `App.vue` the lint is broken (I have Prettier that runs on save). If I run the lint service of the template everything is correct though.